pages/portfolio.html

```diff
@@ -1,12 +1,110 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sana's Projects</title>
+    <link href="../styles/style.css" rel="stylesheet">
 </head>
 <body>
-  
+    <header>  
+        <h1>Some of My Favorite Projects at ada </h1>
+        <div class="top">
+            <a href="about.html">About ME!</a>
+            <a href="index.html"> Swiftie Website</a>
+        </div>
+</header>  
+
+
+<div class="card">
```

Comment:
Consider making this a large unordered list. Then you could replace each div with "card" class with list items. and inside each list item you could have a section that has the link, the description header, the description paragraph, another list with the skills. 

Doing a refactor like this makes your HTML more semantic and you can still achieve the styling you need. Having semantic HTML improves accessibility for your site for folks who may use assistive devices (such devices can struggle to navigate a page with lots of non semantic html like divs and spans)